src/terraform/state.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  required_providers {
+    openstack = {
+      source = "terraform-provider-openstack/openstack"
+      version = "1.42.0"
+    }
+
+    local = {
```

Comment:
For what usage did you install this provider ?

Reply:
copy/pasta fail :shrug: